.emacs.d/lisp/code/editorconfig-c.el

```diff
@@ -0,0 +1,27 @@
+;; Enable EditorConfig mode
+(editorconfig-mode +1)
+
+(defun kotct/editorconfig--disable-smart-tabs-if-indent-tabs-mode-disabled (&optional hash)
```

Comment:
Also the `hash` thing sounds fine. Maybe mention that it's only there for compatibility and is ignored in the docstring.